pom.xml

```diff
@@ -316,7 +316,7 @@ under the License.
     <dependency>
       <groupId>org.apache.maven.wagon</groupId>
       <artifactId>wagon-http-lightweight</artifactId>
-      <version>1.0-beta-6</version>
+      <version>3.3.4</version>
```

Comment:
Does this incur any class hanges?

Reply:
In 3.3.4 this is three classes in org.apache.maven.wagon.providers.http

In 1.0-beta-6 this is two classes in the same package, so as best I can tell the answer is no. 

I'm not 100% sure we need this dependency at all. We don't seem to import any of those, though it might get loaded though some sort of indirect service provider mechanism.

Reply:
also, it's provided so at runtime this gets replaced  anyway.

Reply:
Maven core comes already with a provider. I bet this one is not necessary. Have you tried to remove it and run all ITs?

Reply:
Yes, that's how I got the failure above.

Reply:
This likely applies to tests only because the plugin cannot be used w/o Maven. That's my assumption/understanding.